### PR TITLE
[CDAP-7127] Remove Deprecated Explore Instances Property

### PR DESCRIPTION
**Pull request by knazir**
Issue: https://issues.cask.co/browse/CDAP-7127
Build: http://builds.cask.co/browse/CDAP-DUT4715-2

### CODE REVIEW

**Comment by anwar6953** [comment]
why is the diff containing so many commits, and why are there conflicts?
What branch did you branch off of?

Edit: nvm, you fixed the branch its merging against.

**Comment by knazir** [comment]
@anwar6953 how dare you

**Comment by knazir** [comment]
We should probably change the default branch back to develop now to avoid issues like that?

**Comment by awholegunch** [comment]
Question: when (if ever) do these properties come out of `cdap/cdap-docs/tools/cdap-default/cdap-default-deprecated.xml`?

**Comment by anwar6953** [comment]
@awholegunch I don't think these properties are in cdap-default-deprecated. Or, they shouldn't be.

**Comment by knazir** [comment]
@anwar6953 both properties are in `cdap-default-deprecated` last I checked. Not sure if they're put their automatically by a tool or if they were added. Should they be removed as well?

**Comment by anwar6953** [comment]
I think we can also remove it from cdap-default-deprecated.xml and cdap-default-exclusions.txt

**Comment by anwar6953** [comment]
and from here?
https://github.com/caskdata/cdap/blob/develop/cdap-docs/tools/cdap-default/cdap-default-exclusions.txt

**Comment by knazir** [comment]
@anwar6953, my bad, didn't see that part of your comment.

**Comment by awholegunch** [comment]
@knazir It's actually not essential that they come out of the _exclusions_ list, as that way they will always be ignored. But it's also fine to clean them up.

File: cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java [hunk 1]
```diff
@@ -817,15 +817,11 @@
     /** Twill Runnable configuration **/
     public static final String CONTAINER_VIRTUAL_CORES = "explore.executor.container.num.cores";
     public static final String CONTAINER_MEMORY_MB = "explore.executor.container.memory.mb";
-    public static final String CONTAINER_INSTANCES = "explore.executor.container.instances";
```
Comment (awholegunch) [inline cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java:820]: `explore.executor.container.instances` and `explore.executor.max.instances` are also in `~/Source/cdap/cdap-ui/server/config/development/cdap.json`

I'm guessing they should also come out of there, but @ajainarayanan should confirm.

Comment (elfenheart) [reply cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java:820]: @awholegunch that file doesn't really matter, it's for development purpose only

Comment (awholegunch) [reply cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java:820]: Still, nine months from now, someone will ask on "Slack": 

```
anyone remember why these are here???
```

Comment (anwar6953) [reply cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java:820]: +1 to removing everything except what's needed from that file.

**Comment by knazir** [comment]
@Removed from `cdap.json` @anwar6953 @awholegunch

**Comment by awholegunch** [comment]
LGTM

**Comment by knazir** [comment]
much thanks @awholegunch guild is breen: http://builds.cask.co/browse/CDAP-DUT4715-3 going to squash and merge

**Pull request merged**